### ISSUE TITLE
"Browse Notes" layer should be labeled "See/Add error reports" and be explained

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#4901]
**[Submitted to the original trac issue database at 6.26pm, Wednesday, 17th July 2013]**

Having passed it by without noticing what it means, I think that "Browse Notes" layer should be labeled "See / Add error reports". Also at the bottom of the page. Especially since [http://wiki.openstreetmap.org/wiki/Notes its wiki page] states that it's for reporting errors (or lacks) and not adding various kinds of notes. [[BR]]
Also, a link to an explanation page made for the average user would be very welcome.  Having helped clearing OpenStreeBug, I can testify how non obvious what to say is to some reporters.  Even accustomed ones: I just changed [http://wiki.openstreetmap.org/wiki/Notes#Adding_a_note in the wiki page] the phrase saying that "oneway in wrong direction" is a good example.
Having the pedestrian be in the mapper's mind for a while is challenging.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 6.43pm, Wednesday, 17th July 2013]**

The name was deliberately changed to notes, after long deliberation, to reflect that fact that "error reports" is not the only use.

**Comment by tomhughes** [comment]
**Author: Papou**
**[Added to the original trac issue at 6.56pm, Thursday, 18th July 2013]**

That may be true but the result is that the reader does not know what it's all about.
It's extremely surprising that you "won't fix" the total absence of information with the link I suggest.
Expect a 3-way chat system.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 7.56pm, Thursday, 18th July 2013]**

What do you mean by a three way chat system?

As far as providing more explanation goes it was also a deliberate decision to keep the description of what to do to a minimum because a causal reporter is not going to go and read a long essay before entering something - if they see a massive block of rules they're just going to forget it and go and do something else.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 7.58pm, Thursday, 18th July 2013]**

I would just also add that of course, if as we gather experience these things prove to be a problem then we will change them, but we deliberately decided to make things as simple as possible initially and to then see what happens and learn from experience where any problems might be.

**Comment by tomhughes** [comment]
**Author: Richard**
**[Added to the original trac issue at 10.02pm, Thursday, 18th July 2013]**

Although it's perfectly clear in British English, it's possible that the word isn't clear in other languages (such as French or the bizarre perversion that is sometimes known as American "English"). In which case you can of course use OSM's existing translation mechanisms to substitute a word or phrase which is more understandable in your language. That is kind of the point of the translation system.

**Comment by tomhughes** [comment]
**Author: emacsen**
**[Added to the original trac issue at 12.51am, Friday, 19th July 2013]**

Richard, while I certainly can't speak for British English speakers, I can confirm that it's confusing to several of us Americans, as I've already cleared out more than one note that a user created saying something like "We'll meet here".

I think perhaps "error" is too strong a word, so maybe "Suggest an improvement" might be better, as it's neutral and covers both errors as well as enhancements.

Baring that, maybe we can create some UI element saying something like "What's this?" and when you hover over it, it explains what a note is, how we use it, etc.

**Comment by tomhughes** [comment]
**Author: Richard**
**[Added to the original trac issue at 6.16am, Friday, 19th July 2013]**

Indeed, conversations with you are why I suggested an American "English" translation might be helpful!

**Comment by tomhughes** [comment]
**Author: Papou**
**[Added to the original trac issue at 1.09pm, Wednesday, 24th July 2013]**

Is it possible to be warned by e-mail about being commented? Thanks.

If you say you "wontfix" the lack of user education, then the mapper "wontfix" the errors with lack of information too, he won't start an investigation. [http://www.papou.byethost9.com/notes/OSM/ I have found this personal scratched text] very useful to get more details. If you call that a long essay and a massive lot of rules, we have a different definition of the word "necessary" there. I have just added topic 1 to make the text usable before "noting" instead of just when asking for more information.
